docs/content/en/contribute/docs/_index.md

```diff
@@ -11,3 +11,5 @@ but all material here has been reviewed for technical accuracy.
 Keptn software and documentation
 is developed and maintained by the Keptn community.
 We invite you to join us.
+
+## [Linters Requirements](docs/content/en/contribute/docs/linters-requirements/_index.md)
```

Comment:
this link seems to be not correct - 
```suggestion
## [Linters Requirements](./linters-requirements/_index.md)
```

and it is still a link within a heading which we try to avoid